### DISCUSSION TITLE
游戏内无法识别皮肤

### DISCUSSION THREAD

**Discussion by Michael66625** [Meloong-Git/PCL#573] [提问] [answered]
在官方启动器内设置的皮肤，在游戏内会被修改为默认皮肤。

**Comment by WForst-Breeze** [comment, marked as answer]
您的意思是说，您在官启中设置完皮肤再去PCL2中登录发现皮肤仍为默认皮肤吗？你可以换完皮肤等几分钟（往往这样就可以解决了）再登陆试试或换个启动器试试问题是否出现，如问题**频繁出现**，或许你可以提交 [**综合 Bug 反馈**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+Bug%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=bug9.yml) 并上传详细信息

**Reply by WForst-Breeze** [reply]
如果你指的是离线皮肤，看看是不是 #590 的问题

**Reply by Michael66625** [reply]
![Snipaste_2022-12-16_15-46-59.png](https://user-images.githubusercontent.com/79311405/208066539-264af757-5beb-46ac-8756-afe1e32f3c62.png)![Snipaste_2022-12-16_15-44-51.png](https://user-images.githubusercontent.com/79311405/208066549-cb0bf409-f74c-41f4-82b7-9117e2115a38.png)

**Reply by WForst-Breeze** [reply]
使用其他启动器是否出现该问题？其他版本是否出现该问题？qwq

**Reply by Michael66625** [reply]
已解决，突然就好了，其它启动器没有这个问题。

**Comment by ilgnefz** [comment]


**Reply by Michael66625** [reply]
不是离线模式，并且提示皮肤刷新完成，进入游戏后，仍然显示为默认皮肤。

**Comment by Michael66625** [comment]
> 您的意思是说，您在官启中设置完皮肤再去PCL2中登录发现皮肤仍为默认皮肤吗？你可以换完皮肤等几分钟（往往这样就可以解决了）再登陆试试或换个启动器试试问题是否出现，如问题**频繁出现**，或许你可以提交 [**综合 Bug 反馈**](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+Bug%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=bug9.yml) 并上传详细信息
在离线模式以及非离线模式下均存在此问题，皮肤仍显示为默认皮肤。
